customisation/parallel-data.md

```diff
@@ -21,8 +21,7 @@ Parallel data sets can be created manually, automatically, or created synthetica
 - Human [post-editing](../workflows/post-editing.md)
 - [Crawling](crawling.md)
 - [Alignment](alignment.md)
-
-Parallel data can be created by crawling and aligned monolingual test, and by [back-translation](back-translation.md) or [back-copying](back-translation.md).
```

Comment:
I suggest removing the whole sentence here, as it mostly repeats the previously mentioned points. As for creating parallel data by aligned monolingual text, I'm not sure if it's relevant here, as monolingual data alignment is used to create comparable corpora in a single language.

Reply:
Yes, you are right!